toolkit/tools/imagegen/configuration/parse_partition.go

```diff
@@ -71,6 +71,33 @@ func populatepartCmdProcessMap() {
 	partCmdProcess[kickstartPartitionFsType] = processPartitionFsType
 }
 
+func processPartitionTableType() (err error) {
+	// In kickstart installation scenario, the partition table type is set to
+	// MBR by default. The Anaconda installer has this config "--gpt" that indicates
+	// whether the users prefer creation of GPT disk label or not. The value of "--gpt"
+	// is a bool where "True" indicates using GPT and "False" if not, which means using MBR.
+	// This config is set as a boot option within /proc/cmdline, which will be parsed by anaconda
+	// during installation process. Thus, Mariner will also pick the same design to reach compatibility
+	// with kickstart scenario
+
+	// Please note that this code is only executed during kickstart installation, when "IsKickStartBoot" is set to true.
```

Comment:
Do we want an explicit check in this function for this condition that `IsKickStartBoot`=`true`?

Reply:
We already checked this condition inside imager.go: https://github.com/microsoft/CBL-Mariner/blob/e3174308e7fa883c3dc52182da13c66355607aea/toolkit/tools/imager/imager.go#L78. 

This whole parser module can only be triggered if **IsKickStartBoot** is set to true